roxie/ccd/ccdmain.cpp

```diff
@@ -717,6 +717,20 @@ int STARTQUERY_API start_query(int argc, const char *argv[])
         if (udpSnifferEnabled && !roxieMulticastEnabled)
             DBGLOG("WARNING: ignoring udpSnifferEnabled setting as multicast not enabled");
 
+        int ttlTmp = topology->getPropInt("@multicastTTL", 1);
```

Comment:
Added to environment.xml.in, thanks.
Should I make multicastTTL signed ?  Or just make neg values equal to 255 ?